01-arch/09-erasure-coding.md

```diff
@@ -0,0 +1,81 @@
+## Erasure Coding
+
+NeoFS applies erasure coding (further EC) to user data for its protection. Being an alternative to replication, EC also
+provides fault tolerance through data recovery. The distinctive feature is storage efficiency: EC out-turn uses less storage
+space compared to simply creating full copies of the data.
```

Comment:
for the same reliability requirements in terms of number of parts that can be lost without affecting object availability?